core/core_modules/kpm.js

```diff
@@ -43,14 +43,18 @@ var git = require.once('../git.js'),
         install: {
             run: function(args, api, event) {
                 if (args.length === 0) {
-                    api.sendMessage('No modules provided to install!', event.thread_id);
+                    api.sendMessage('Nothing provided to install!', event.thread_id);
                     return;
                 }
 
                 for (var i = 0; i < args.length; i++) {
                     var url = args[i];
-                    if (url.startsWith('http') || url.startsWith('ssh')) {
-                        install.call(this, url, api, event);
+                    if (url.startsWith('ssh') || url.endsWith('.git')) {
+                        gitInstall.call(this, url, api, event);
+                        continue;
+                    }
+                    else if (url.startsWith('http') && (url.endsWith('.coffee') || url.endsWith('.js'))) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["scriptInstall" is not defined.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529976242/issues/source?bid=3282987&fileBranchId=3282998#l57)